Cython/Utility/ModuleSetupCode.c

```diff
@@ -1775,7 +1781,8 @@ static void __Pyx_FastGilFuncInit0(void) {
 #else
 
 static void __Pyx_FastGilFuncInit0(void) {
-  CYTHON_UNUSED void* force_use = (void*)&__Pyx_FetchCommonPointer;
+  void* force_use = (void*)&__Pyx_FetchCommonPointer;
+  CYTHON_UNUSED_VAR(force_use);
```

Comment:
This can probably be simplified to this:
```suggestion
  CYTHON_UNUSED_VAR(&__Pyx_FetchCommonPointer);
```